plugins/auto-sizes/tests/test-auto-sizes.php

```diff
@@ -206,13 +206,11 @@ public function data_image_with_existing_auto_sizes(): array {
 	}
 
 	/**
-	 * Test printing the meta generator tag.
-	 *
-	 * @covers ::auto_sizes_render_generator
+	 * Test auto-sizes meta tag present in wp_head hook.
 	 */
-	public function test_auto_sizes_render_generator(): void {
-		$tag = get_echo( 'auto_sizes_render_generator' );
-		$this->assertStringStartsWith( '<meta', $tag );
+	public function test_auto_sizes_meta_tag_present_in_wp_head_hook(): void {
+		$tag = get_echo( 'wp_head' );
+		$this->assertStringContainsString( '<meta', $tag );
```

Comment:
I think this change can be reverted as well.

Reply:
Either way works for me. With `wp_head`, it's more of an integration test, with `auto_sizes_render_generator`, it's more of a unit test.

Since we already test the action hook above, I slightly lean towards reverting as Weston suggested.

Reply:
Reverted in https://github.com/WordPress/performance/pull/1547/commits/791c5a2ec412cdbb22532b9304075e494c3d1e08